Makefile

```diff
@@ -1708,6 +1710,80 @@ frwiki_tuning_reports: \
 	tuning_reports/frwiki.goodfaith.md
 
 
+############################# Galician Wikipedia ################################
+datasets/glwiki.human_labeled_revisions.60k_2019.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/glwiki/85/ > $@
+
+
+datasets/glwiki.sampled_revisions.60k_2019.json:
+	wget -qO- https://quarry.wmflabs.org/run/385851/output/0/json-lines?download=true > $@
+
+datasets/glwiki.autolabeled_revisions.60k_2019.json: \
+		datasets/glwiki.sampled_revisions.60k_2019.json
+	cat $< | \
+	./utility autolabel --host=https://gl.wikipedia.org \
+		--trusted-groups=checkuser,bureaucrat,sysop,eliminator,bot \
+		--trusted-edits=1000 \
+		--revert-radius=5 \
+		--verbose > $@
+
+datasets/glwiki.labeled_revisions.60k_2019.json: \
+		datasets/glwiki.human_labeled_revisions.60k_2019.json \
```

Comment:
This human_labeled_revisions dataset shouldn't be here.  It shouldn't have been auto-generated.  But if it was, we might have a bug in our Makefile template.

Reply:
I actually added this manually because `datasets/glwiki.labeled_revisions.w_cache.60k_2019.json` was calling this but it did not exist. 

This is actually due to an incorrect config, I'll push a fix up soon.